spec/built-in-tiles-spec.coffee

```diff
@@ -48,6 +48,65 @@ describe "Built-in Status Bar Tiles", ->
         runs ->
           expect(fileInfo.currentPath.textContent).toBe 'sample.txt'
 
+    describe "when buffer's path is clicked", ->
+      it "copies the absolute path into the clipboard if available", ->
+        waitsForPromise ->
+          atom.workspace.open('sample.txt')
+
+        runs ->
+          fileInfo.currentPath.click()
+          expect(atom.clipboard.read()).toBe fileInfo.getActiveItem().getPath()
+
+    describe "when path of an unsaved buffer is clicked", ->
+      it "copies the 'untitled' into clipboard", ->
+        waitsForPromise ->
+          atom.workspace.open()
+
+        runs ->
+          fileInfo.currentPath.click()
+          expect(atom.clipboard.read()).toBe 'untitled'
+
+    describe "when buffer's path is not clicked", ->
+      it "doesn't display a path tooltip", ->
+        jasmine.attachToDOM(workspaceElement)
+        waitsForPromise ->
+          atom.workspace.open()
+
+        runs ->
+          expect(document.querySelector('.tooltip')).not.toExist()
+
+    describe "when buffer's path is clicked", ->
+      it "displays path tooltip and the tooltip disappears after 2 seconds", ->
+        jasmine.attachToDOM(workspaceElement)
+        waitsForPromise ->
+          atom.workspace.open()
+
+        runs ->
+          fileInfo.currentPath.click()
+          expect(document.querySelector('.tooltip')).toBeVisible()
+          advanceClock(2000)
```

Comment:
Since `setTimeout` is not exact to the millisecond, I'd probably advance the clock just a little longer here. Just to make sure the test won't randomly fail if the tooltip is there for a few ms longer. What do you think?